src/error.rs

```diff
@@ -188,7 +188,7 @@ pub enum AxoassetError {
         dest_path: String,
         /// Details of the error
         #[source]
-        details: std::io::Error,
+        details: Option<std::io::Error>,
```

Comment:
is there a case where we want this to be none? i think ideally, this always contains details